src/program/example/example.lua

```diff
@@ -0,0 +1,17 @@
+module(..., package.seeall)
+
+local ffi = require("ffi")
+local C = ffi.C
+local lib = require("core.lib")
+local usage = require("program.example.README_inc")
+
+function run (args)
```

Comment:
This code for dispatching on a subcommand (`sprayer <subcommand> ...`) is a bit tricky as an introduction to Snabb Switch. I think that we should simplify this somehow.

Some ideas:

* Stop using subcommands and simply flatten out the command-name space. (s/`snabbnfv traffic/snabbnfv-traffic/`)
* Standardize (move) subcommand logic somewhere else (e.g. `main.lua` or `lib.lua`) to make the example program simpler.
* Change the example to use a different command-line syntax (e.g. `-r <file>` for replay and `-s <file>` for spray).